sensors:
  - sensor_type: point_cloud_sensor
    point_cloud_topic: /camera/rgb/points
    max_range: 5.0
    frame_subsample: 1
    point_subsample: 1

